documentation/List-of-Action-plugins.md

```diff
@@ -2,59 +2,259 @@
+
+Creates a new Organisation in the database based on user input parameters.
+
+- _Action Code:_ **`createOrg`**
+
+| Input parameters<br />(\*required) <br/> | Output properties |
+| ---------------------------------------- | ----------------- |
+| `name`\*                                 | `orgId`           |
+| `licence_number`                         | `orgName`         |
+| `address`                                |                   |
+
+---
+
+### Join User to Organsation
+
+Creates a link between a user and an organisation -- i.e. user is a "member" of that organisation. Adds a new record to the `user_organisation` table.
+
+- _Action Code:_ **`joinUserOrg`**
+
+| Input parameters<br />(\*required) <br/>    | Output properties |
+| ------------------------------------------- | ----------------- |
+| `user_id`\*                                 | `userOrgId`       |
+| `org_id` \*                                 |                   |
+| `user_role` (Arbitrary title, e.g. "Owner") |                   |
+
+---
+
+### Grant Permissions
+
+Grants permission to user/org -- i.e. creates `permission_join` from user/org to permission name. If org not provided, the permission will be granted to the user only.
+
+- _Action Code:_ **`grantPermission`**
+
+| Input parameters<br />(\*required) <br/> | Output properties   |
+| ---------------------------------------- | ------------------- |
+| `username`\*                             | `permissionJoinIds` |
+| `orgName`                                | `permissionNames`   |
+| `permissionNames`\* [Array of names]     |                     |
+
+---
+
+### Generate Review Assignments
+
+Generates records in the `review_assignment` table -- i.e. which users (reviewers) are allowed to do a review for the current stage/level (and for which Sections). The records are set with status "Available" or "Available for self-assignment", waiting for assignment.
+
+It also creates records in the `review_assignment_assigner_join` table -- basically a list of users who have permission to make the _assignments_ in the review_assignment table.
+
+Should be run whenever an application or review is submitted, and it will generate the review assignments for the next _review_ that should be done.
+
+Note: if assignment records already exist (i.e. if it's a re-assignment), they will just be updated (with a new timestamp)
+
+_TO-DO: decide what to do with records that exist, but no longer should be (e.g. the reviewer has been removed/permissions revoked)_
+
+- _Action Code:_ **`generateReviewAssignments`**
+
+| Input parameters<br />(\*required) <br/> | Output properties                 |
+| ---------------------------------------- | --------------------------------- |
+| `applicationId`\*                        | `reviewAssignmentIds`             |
+| `reviewId`                               | `reviewAssignmentAssignerJoins`   |
+| `templateId`\*                           | `reviewAssignmentAssignerJoinIds` |
```

Comment:
`templateId` can also be removed

Reply:
Cool. I also went an combined the Notes and added a bit more in the "TO-DO" section.